internal/ethapi/api.go

```diff
@@ -1170,16 +1166,18 @@ func DoEstimateGas(ctx context.Context, b Backend, args TransactionArgs, blockNr
 	} else {
 		feeCap = common.Big0
 	}
+
+	state, header, err := b.StateAndHeaderByNumberOrHash(ctx, blockNrOrHash)
+	if state == nil || err != nil {
+		return 0, err
+	}
+	err = overrides.Apply(state)
+	if err != nil {
```

Comment:
```suggestion
	if err := overrides.Apply(state); err != nil {
```
I nkow you just lifted it up, but still...

Reply:
done